test/distributed/org/apache/cassandra/distributed/test/MessageFiltersTest.java

```diff
@@ -63,17 +66,17 @@ public void simpleOutboundFiltersTest()
 
     private static void simpleFiltersTest(boolean inbound)
     {
-        int VERB1 = Verb.READ_REQ.id;
```

Comment:
if trying to remove the import of Verb, can you document these numbers?  1, 2, and 3 are less readable than before

Reply:
reverted to prev version